scripts/start/runGamsCompile.R

```diff
@@ -0,0 +1,38 @@
+#' compile a modelFile with a cfg 
+#'
+#' @param modelFile filename of model file to be compiled
+#' @param cfg list with REMIND configuration
+#' @param interactive boolean, if TRUE, will ask user to compile again after fails
+#' @author Oliver Richters
+#' @return boolean whether compilation was successful
+runGamsCompile <- function(modelFile, cfg, interactive = TRUE) {
```

Comment:
I remember that @0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q suggested that `--gamscompile` should run `prepare()` to download input data. Can this easily be done now, or would this be a bigger change?

Reply:
I think it would be rather easy, but let's do it in the next restructuring PR :wink: